### DISCUSSION TITLE
Default color for non colored tags

### DISCUSSION THREAD

**Discussion by RoinujNosde** [Tomut0/SimpleClansStats#20] [Ideas]
![image](https://user-images.githubusercontent.com/6441328/113946504-faba9d80-97de-11eb-8b38-cabfd6dda0ae.png)
MIB seems off.
Non colored tags could be gray or some color from .env

**Comment by Tomut0** [comment]
Can you show me how's MIB looks in web inspector?

**Comment by RoinujNosde** [comment]
![image](https://user-images.githubusercontent.com/6441328/113947517-32c2e000-97e1-11eb-84cd-a5f2aa4f2972.png)
